lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -0,0 +1,34 @@
+<%page args="course_program_info, enrollment_mode" />
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<%namespace name='static' file='../static_content.html'/>
+<div class="message message-status is-shown credit-message">
+    <div class="xseries-action">
+        <div class="message-copy xseries-msg">
+            <p>
+                <b class="message-copy-bold">${_("XSeries Program: Interested in more courses in this subject?")}</b>
+            <p>
+            <p class="message-copy">
+                ${_("This course is 1 of {course_count} courses in the {link_start}{course_display}{link_end}.").format(
```

Comment:
I see we're hardcoding XSeries elsewhere in the template. Should this instead be:
```html
${_("This course is 1 of {course_count} courses in the {link_start}{course_display}{link_end} XSeries.").format(
```